docs/manual/scala/guide/cluster/SerializationPlayJson.md

```diff
@@ -0,0 +1,162 @@
+## Play-JSON Serialization
+
+To enable JSON Serialization there are three steps you need to follow.
+
+The first step is to define your [Format](https://www.playframework.com/documentation/2.6.x/api/scala/play/api/libs/json/Format.html) for each class that is to be serialized, this can be done using [automated mapping](#Automated-mapping) or [manual mapping](#Manual-mapping).
```

Comment:
How about using the latest version of Play for the link to Scaladoc?  
Here and in a couple of other places. 😄

Reply:
Good call but we're still not addressing that. I'll do a single pass at the very last moment before publishing the GA release. 

Hmm, I guess I started a DRAFT PR already as a reminder: https://github.com/lagom/lagom/pull/2409